build-tools/config/webpack/webpack.partial.conf.plugins.js

```diff
@@ -108,6 +107,22 @@ module.exports = ({ config, isDevelopment, buildType }) => webpackConfig => {
         }),
       );
     }
+    if (config.enablePrerender) {
+      plugins.push(
+        new PrerenderSPAPlugin({
+          staticDir: path.join(config.projectRoot, 'dist'),
+          // Add routes to prerender.
+          routes: ['/', '/about'],
+          renderAfterDocumentEvent: 'x-app-render',
+          postProcess: route => {
+            route.html = route.html // eslint-disable-line
```

Comment:
disable should always include the rule you're disabling

Reply:
This is not needed either.